ff-api-update.php

```diff
@@ -1,32 +1,73 @@
 <?php
- 
+
 /* 
  * refresh the number of active nodes in the freifunk fulda community.
  * the node information is gathered from the ffmap nodes.json file and
  * written to the freifunk.net API file.
  */
- 
-$NODELIST = 'https://meshviewer.fulda.freifunk.net/data/nodes.json';
-$APIFILE = '/path/to/FreifunkFulda-api.json';
- 
-$map_json = file_get_contents($NODELIST);
+
+# disable SSL verification to prevent errors
+$arrContextOptions=array(
+    "ssl"=>array(
+        "verify_peer"=>false,
+        "verify_peer_name"=>false,
+    ),
+);
+
+# json validation
+function validateJson($json) {
+	$json_test = json_decode($json);
```

Comment:
Let validate return the parsed data and use it instead of a swcond ccall to parse.